### PR TITLE
Code review rvrs.load vip veteran other atr pr

### PR DESCRIPTION
**Pull request by sailendrasingh1**
ITEM 153
Procedure: RVRS.Load_VIP_VeteranOtherAtrPr

### CODE REVIEW

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+
+	
+PRINT '6'  + CONVERT (VARCHAR(50),GETDATE(),109)	
+					
+   
+		SELECT *
+			
+	        ,CASE WHEN VeteranEntries>3 THEN 'VeteranEntries|Error:Not a valid Veteran Entry' ELSE '' END AS LoadNote_1
+	        ,CASE WHEN VeteranEntries = 0 AND (WAR1 IS NOT NULL OR WAR2 IS NOT NULL OR WAR3 IS NOT NULL) THEN 'VeteranEntries|Warning:Veteran Entry and War Entered Mismatch' ELSE '' END AS LoadNote_2
+	        ,CASE WHEN VeteranEntries = 1 AND (WAR1 IS NULL OR WAR2 IS NOT NULL OR WAR3 IS NOT NULL) THEN 'VeteranEntries|Warning:Veteran Entry and War Entered Mismatch' ELSE '' END AS LoadNote_3
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql:225]: SSA

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+	        ,CASE WHEN VeteranEntries = 2 AND (WAR1 IS NULL OR WAR2 IS NULL OR WAR3 IS NOT NULL) THEN 'VeteranEntries|Warning:Veteran Entry and War Entered Mismatch' ELSE '' END AS LoadNote_4
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql:226]: SSA

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql [hunk 3]
```diff
@@ -0,0 +1,466 @@
...
+	        ,CASE WHEN VeteranEntries = 3 AND (WAR1 IS NULL OR WAR2 IS NULL OR WAR3 IS  NULL) THEN 'VeteranEntries|Warning:Veteran Entry and War Entered Mismatch' ELSE '' END AS LoadNote_5
+
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql:228]: SSA

**Review by almabekwoldesenbet — CHANGES_REQUESTED** [review]
Minor changes - you can create Prod Version with the requested fixes

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql [hunk 4]
```diff
@@ -0,0 +1,466 @@
...
+*/
+
+	
+PRINT '6'  + CONVERT (VARCHAR(50),GETDATE(),109)	
+					
+   
+		SELECT *
+			
+	        ,CASE WHEN VeteranEntries>3 THEN 'VeteranEntries|Error:Not a valid Veteran Entry' ELSE '' END AS LoadNote_1
+	        ,CASE WHEN VeteranEntries = 0 AND (WAR1 IS NOT NULL OR WAR2 IS NOT NULL OR WAR3 IS NOT NULL) THEN 'VeteranEntries|Warning:Veteran Entry and War Entered Mismatch' ELSE '' END AS LoadNote_2
```
[the remark this answers is not in the document]
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql:224]: added DimWarId in the header section

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql (hunk 1 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql:225]: added DimWarId in the header section

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql (hunk 2 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_VeteranOtherAtrPr.sql:226]: added DimWarId in the header section

**Review by almabekwoldesenbet — APPROVED** [review]
looks good now

**Pull request merged**